js/login.js

```diff
@@ -0,0 +1,123 @@
+const MASTER_ACCOUNT = {
+  EMAIL: 'test@codeit.com',
+  PASSWORD: 'codeit101'
+}
+
+const REGEX = {
+  EMAIL: /^[0-9a-zA-Z]([-_\.]?[0-9a-zA-Z])*@[0-9a-zA-Z]([-_\.]?[0-9a-zA-Z])*\.[a-zA-Z]{2,3}$/i,
+  PASSWORD: '',
+}
+
+const ERROR_MSG = {
+  BLANK: {
+    EMAIL: '이메일을 입력해 주세요.',
+    PASSWORD: '비밀번호를 입력해 주세요.'
+  },
+  INVALID: {
+    EMAIL: '올바른 이메일 주소가 아닙니다.'
+  },
+  INCORRECT:{
+    EMAIL: '이메일을 확인해 주세요.',
+    PASSWORD: '비밀번호를 확인해 주세요.'
+  }
+}
+
+Object.freeze(MASTER_ACCOUNT);
+Object.freeze(REGEX);
+Object.freeze(ERROR_MSG);
+
+const $ = (selector, element = document) => element.querySelector(selector);
+const isMaster = (email, password) => email === MASTER_ACCOUNT['EMAIL'] && password === MASTER_ACCOUNT['PASSWORD'];
+const isEmail = (email) => REGEX['EMAIL'].test(email);
```

Comment:
객체에 접근할때 변수나, 예약어등을 사용하지 않고 일반적인 문자열을 사용하는 경우에는 .(dot notation)으로 접근하는 방식을 일반적으로는 더 많이 쓰는 것 같아요!

```suggestion
const isMaster = (email, password) => email === MASTER_ACCOUNT.EMAIL && password === MASTER_ACCOUNT.PASSWORD;
const isEmail = (email) => REGEX.EMAIL.test(email);
```